diagnostic.md

```diff
@@ -35,21 +35,17 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+It is an absolute path because it starts at the root directory.
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
- <!-- Answer Starts Here -->
-
- <!-- Answer Ends Here -->
+We would use a relative path because we are referring to a location on the
```

Comment:
In fact, it would be an absolute. Reason being, you need the root path (url) then navigate down from there to access the font.